app/models/form_template.rb

```diff
@@ -1,17 +1,8 @@
 # frozen_string_literal: true
 
 class FormTemplate < ActiveRecord::Base
-  validates :name,
-            presence: true,
-            uniqueness: true,
-            length: {
-              maximum: SiteSetting.max_form_template_title_length,
-            }
-  validates :template,
-            presence: true,
-            length: {
-              maximum: SiteSetting.max_form_template_content_length,
-            }
+  validates :name, presence: true, uniqueness: true
+  validates :template, presence: true
   validates_with FormTemplateYamlValidator
```

Comment:
btw, this custom validator fails when the template is nil:

```
TypeError: no implicit conversion of nil into String
```

maybe we should run it only when there is a template to be validated?

```suggestion
  validates_with FormTemplateYamlValidator, if: ->(ft) { ft.template }
```